frontstage/views/surveys/upload_survey_failed.py

```diff
@@ -33,6 +33,9 @@ def upload_failed(session):
     elif error_info == "size":
         error_info = {'header': "Error uploading - file size too large",
                       'body': 'The spreadsheet must be smaller than 20MB in size'}
+    elif error_info == "sizeSmall":
+        error_info = {'header': "Error uploading - file size too small",
+                      'body': "The spreadsheet must be larger than 6KB in size"}
```

Comment:
Is there any particular reason we've gone for 6KB here?

Reply:
the template that is given is 6kb, so i used that. however it is worth noting that the actual limit is 1 byte, just so frontstage rejects 0 byte files. Anything above zero can go through.

Reply:
I think it should be 1kb, if thats what the actual limit is